Core/include/Acts/Vertexing/HelicalTrackLinearizer.ipp

```diff
@@ -132,69 +130,75 @@ Acts::Result<Acts::LinearizedTrack> Acts::
     // Fifth row
     momentumJacobian(4, 2) = 1.;
 
-    // TODO: Derivatives of time --> Next PR
+    // Sixth row
+    momentumJacobian(5, 0) = -d0 / betaT;
   } else {
     // Helix radius
-    double rho{sinTheta * (1. / qOvP) / Bz};
+    ActsScalar rho{sinTheta * (1. / qOvP) / Bz};
```

Comment:
```suggestion
    ActsScalar rho = sinTheta * (1. / qOvP) / Bz;
```
Is there a strong reason not to use the same initialization like in the rest of the file?

Reply:
No , I will change it! Is there a preferred way of initializing a variable upon its declaration in ACTS? I read that curly braces are preferred in general, but it might be inconsistent to start using them now..

Reply:
No, we don't have a standard I think, but I had the feeling that the assignment operator `=` is more common in Acts and I can read it more easily.